authorization-server/src/modules/auth/auth.controller.ts

```diff
@@ -267,10 +267,14 @@ export class AuthController {
   }
 
   private unsetAuthCookies(res: Response) {
+    const { sameSite } = this.authService.getAuthCookiesOptions();
```

Comment:
Why just `sameSite`? It seems to me that we need to set `secure` as well for example?

Reply:
Good question. I am not sure how it actually behaves when having individual options set to other values here than when setting a cookie, so I decided to set only that that is problematic.

Reply:
I think that other flags are irrelevant when unsettling a cookie. It is removed if not ignored by a browser, and it can be ignored only when the `sameSite` flag is causing it.